mantle/platform/conf/conf.go

```diff
@@ -332,6 +332,76 @@ func (c *Conf) MergeV34exp(newConfig v34exptypes.Config) {
 	c.ignitionV34exp = &mergeConfig
 }
 
+// Merge all configs into a V3.4 config
+func MergeUserData(userDataObjs []*UserData) (*UserData, error) {
+	config := Conf{}
+	config.ignitionV34exp = &v34exptypes.Config{
+		Ignition: v34exptypes.Ignition{
+			Version: "3.4.0-experimental",
```

Comment:
Why use experimental config here instead of just the latest stable config?

Reply:
Ah sorry about that! I left a little note in the commit message but I should've also added a comment. I used V3.4 because `ParseCompatibleVersion`  parses configs 3.0 to 3.4 as opposed to in V3.3 we would need to recognize a V3.4 config and raise an error. I thought for the initial draft V3.4 was a little more simple. For the final PR, I will use the latest stable config.

Reply:
ok ✔️

Reply:
Hmm, I think using the latest experimental config is the only sane choice if we want to support non-exclusive tests which make use of it.